packages/core/package.json

```diff
@@ -46,6 +46,7 @@
     "@rsbuild/core": "^1.3.4",
     "birpc": "2.3.0",
     "chai": "^5.2.0",
+    "jest-diff": "^29.7.0",
```

Comment:
`jest-diff`'s dependency is a bit messy https://npmgraph.js.org/?q=jest-diff and it hasn't been maintained by two years (so does Jest). Do we have any alternatives? 👀
<img width="358" alt="image" src="https://github.com/user-attachments/assets/61291461-778c-453c-93a8-ec804efe0fb5" />

Reply:
![image](https://github.com/user-attachments/assets/87e91c15-9349-48ad-a6dd-ce042f151b58)

29.7.0 is the latest version of `jest`. `jest-diff` is used by jest officially and is relatively reliable and stable. And it seems that jest has been preparing for 30.0.0.

Reply:
Not so much maintenance but I think the package is relativly stable https://github.com/jestjs/jest/commits/main/packages/jest-diff. The concern is the bloat of indirect dependence. It's not the most important issue for us now, we might change it in the future.